.cypress/integration/trace_analytics_test/trace_analytics_services.spec.js

```diff
@@ -95,23 +97,27 @@ describe('Testing service view', () => {
         win.sessionStorage.clear();
       },
     });
+    cy.get("[data-test-subj='indexPattern-switch-link']").click();
+    cy.get("[data-test-subj='data_prepper-mode']").click();
     setTimeFilter();
     cy.get('input[type="search"]').first().focus().type(`${SERVICE_NAME}`);
     cy.get('[data-test-subj="superDatePickerApplyTimeButton"]').click();
     cy.get('.euiTableRow').should('have.length.lessThan', 3);//Replaces wait
-    cy.get('[data-test-subj="service-link"]').eq(0).click();
+    cy.get('*[data-test-subj^="service-flyout-action-btntrace_service"]').click();
+    cy.get('[data-test-subj="ActionContextMenu"]').click();
```

Comment:
For elements like `'[data-test-subj="ActionContextMenu"]'` that are reused in multiple places, we could centralize them in a utility file with exports. This way, any changes to the selector would only need to be updated in one place, avoiding the need to modify all occurrences across test classes.

Reply:
export const expandServiceView = (rowIndex = 0) => {
  cy.get('*[data-test-subj^="service-flyout-action-btntrace_service"]').eq(rowIndex).click();
  cy.get('[data-test-subj="ActionContextMenu"]').click();
  cy.get('[data-test-subj="viewServiceButton"]').click();
};

Added this function to utils/constants.js